### PR TITLE
The update that also changed the world

### PR DESCRIPTION
**Pull request by Chocohead**
To both commiserate New Zealand losing to England in the rugby world cup (as promised), and celebrate @Blayyke returning to the MCP scene, have some Intermediaries. They're separate from the 1.2.5 ones as I don't think it would be very likely I could match them directly without getting some of them wrong when there's sets of big changes to knock things about like 1.1 -> 1.2's world format change and 1.8's overhaul of biomes/food/etc.

Should actually all be fine as it but thought it was best you look over everything just in case I missed something.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,6 +43,10 @@ repositories {
 		name "Modmuss50 Repository"
 		url 'https://maven.modmuss50.me'
 	}
+	maven {
+		name "Jitpack"
+		url 'https://jitpack.io/'
```
Comment (liach) [inline build.gradle]: Can we drop this? This makes buildscript slow as hell

File: build.gradle [hunk 2]
```diff
@@ -50,65 +59,58 @@ configurations {
 }
 
 dependencies {
-	enigmaRuntime "net.fabricmc:stitch:0.2.1.61"
-	enigmaRuntime "cuchaz:enigma:0.14.0.128:all"
+	enigmaRuntime 'com.github.Chocohead:Stitch:fc82ff6'
```
Comment (liach) [inline build.gradle]: put this in some directory called `enigmaDep` and declare
```gradle
enigmaRuntime fileTree(dir: "enigmaDep", include: "**.jar")
```

File: build.gradle (hunk 1 above)
Comment (Chocohead) [reply build.gradle]: Just out of curiosity does the newest commit make any noticeable difference?

**Comment by valoeghese** [comment]
Seems legit

**Pull request merged**